integration/test/sanity/resources/verifyLinuxCtlScript.sh

```diff
@@ -48,41 +48,36 @@ assertStatus "cwa_config_status" "configured"
 
 step=3
 /usr/bin/amazon-cloudwatch-agent-ctl -a remove-config -c default -s
-assertStatus "cwa_running_status" "stopped"
-assertStatus "cwa_config_status" "not configured"
-
-step=4
-/usr/bin/amazon-cloudwatch-agent-ctl -a fetch-config -c default -s
 assertStatus "cwa_running_status" "running"
 assertStatus "cwa_config_status" "configured"
 
-step=5
+step=4
 /usr/bin/amazon-cloudwatch-agent-ctl -a prep-restart
 /usr/bin/amazon-cloudwatch-agent-ctl -a stop
 assertStatus "cwa_running_status" "stopped"
 assertStatus "cwa_config_status" "configured"
 
-step=6
+step=5
 /usr/bin/amazon-cloudwatch-agent-ctl -a cond-restart
 assertStatus "cwa_running_status" "running"
 assertStatus "cwa_config_status" "configured"
 
-step=7
```

Comment:
Removing the cwa config and restarting creates a new default config. Since aoc is no longer available. 

Past scenario 1: aoc no cwa run aoc
Past scenario 2: no aoc no cwa run cwa default 

Now: no cwa run cwa default